inc/admin/upgrader.php

```diff
@@ -296,4 +296,10 @@ function secupress_new_upgrade( $secupress_version, $actual_version ) {
 			}
 		}
 	}
+
+	// < 1.2.6.1
```

Comment:
You have this? Since the comment starts with a `<`, you shouldn't have it (I don't).